client/client-multi/client-objective-c/Kaa/KaaClientFactory.m

```diff
@@ -37,11 +41,11 @@ @implementation KaaClientFactory
 }
 
 + (id<KaaClient>)clientWithContext:(id<KaaClientPlatformContext>)context {
-    return [self clientWithContext:context stateDelegate:nil];
+    return [self clientWithContext:context stateDelegate:nil autoGeneratedKeys:NO];
```

Comment:
I'd make the default strategy to generate keys for the user but not expect them to be already generated.